hummingbot/market/bitcoin_com/bitcoin_com_market.pyx

```diff
@@ -266,14 +266,6 @@ cdef class BitcoinComMarket(MarketBase):
             for key, value in saved_states.items()
         })
 
-    async def get_active_exchange_markets(self) -> pd.DataFrame:
```

Comment:
I think we can leave this function as is for now. Deleting it might cause broken references somewhere. Also it seems to be a generic function for most connectors.

Reply:
Thanks, I reverted the commit.